projects/plugins/jetpack/extensions/blocks/google-docs-embed/view.scss

```diff
@@ -0,0 +1,214 @@
+/**
+ * Shared Editor and Front End styles for Google Docs
+ */
+@import "~@wordpress/base-styles/_breakpoints";
```

Comment:
You should be able to use `@automattic/jetpack-base-styles` in this file as well.

Reply:
Sorry, not following what the recommendation is here. Are you suggesting `@automattic/jetpack-base-styles` should be used instead of `~@wordpress/base-styles/_breakpoints`?

Reply:
Sorry, I should have been clearer. Yes, that's it. You can find out more here:
https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/README.md#colors

You can also check other blocks for some examples, e.g.:
https://github.com/Automattic/jetpack/blob/d2d62ec3c350533ae2629e0fec60aa56fc83cc75/projects/plugins/jetpack/extensions/blocks/social-previews/modal.scss#L5
https://github.com/Automattic/jetpack/blob/d2d62ec3c350533ae2629e0fec60aa56fc83cc75/projects/plugins/jetpack/extensions/blocks/social-previews/modal.scss#L71